src/main/java/study/StringCalculator.java

```diff
@@ -0,0 +1,92 @@
+package study;
+
+import java.util.Scanner;
+
+public class StringCalculator {
+
+    public static int plus(int a, int b) {
+        return a+b;
+    }
+
+    public static int minus(int a, int b) {
+        return a-b;
+    }
+
+    public static int multiply(int a, int b) {
+        return a*b;
+    }
+
+    public static int devide(int a, int b) {
+        return a/b;
+    }
+
+    public static int calculate(String calculation, int num, int result){
+        switch (calculation){
+            case "+":
+                result = plus(num,result);
+                break;
+            case "-":
+                result = minus(num,result);
+                break;
+            case "*":
+                result = multiply(num,result);
+                break;
+            case "/":
+                result = devide(num,result);
+                break;
+        }
+        return result;
+    }
+
+    //null값 체크
+    public static void stringCheck(String string){
+        if(string==null||string.equals("")||string.equals(" ")){
```

Comment:
```suggestion
        if(string==null || string.trim().equals("")){
```
`trim`을 사용하면 공백을 없앨 수 있습니다~ 세번째 조건의 경우 한 가지 경우밖에 처리할 수 없어요!